auth-web/src/components/auth/manage-business/AddBusinessDialog.vue

```diff
@@ -191,6 +218,22 @@ export default class AddBusinessDialog extends Vue {
     return CommonUtils.isFirmNumber(this.businessIdentifier)
   }
 
+  get isStaffAccount (): boolean {
+    return this.currentUser.roles.includes(Role.Staff) || false
```

Comment:
This only works for BC Reg staff? You'll have to check to see their ORG if they're in the SBC_STAFF.

Reply:
Here I've changed the code to include `GOVMAccountUser` (sbc staff) for this part.

Reply:
This code is now removed. isStaffAccount is passed from EntityManagement to AddBusinessDialog instead of fetching it here.